Form-Controls/index.html

```diff
@@ -13,15 +13,54 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
-      </form>
+   <fieldset>
```

Comment:
What value is this `<fieldset>` element bringing? What would be worse if you removed it?